### PR TITLE
Feature/25 authentication

### PR DESCRIPTION
**Pull request by eddiejaoude**
* Github OAuth
* Behat 3
* PHPSpec 2

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,68 @@
+{
+    "name": "eaj/myproject",
```
Comment (brettminnie) [inline composer.json]: Did you miss the project name and license here?

Comment (eddiejaoude) [inline composer.json]: Yep :cry:

Comment (brettminnie) [inline composer.json]: Bad Eddie, bad, back to your room :P

Comment (eddiejaoude) [inline composer.json]: Rookie mistake :disappointed:

**Pull request merged**